stage0/supply-chain/config.toml

```diff
@@ -0,0 +1,52 @@
+# cargo-vet config file
+[cargo-vet]
+version = "0.6"
+
+[imports.google]
+url = "https://raw.githubusercontent.com/google/supply-chain/main/audits.toml"
+
+[imports.google.criteria-map]
+crypto-safe = "crypto-safe"
+does-not-implement-crypto = "does-not-implement-crypto"
+
+[imports.mozilla]
+url = "https://raw.githubusercontent.com/mozilla/supply-chain/main/audits.toml"
+
+[policy.bitflags]
+criteria = ["safe-to-deploy", "crypto-safe"]
```

Comment:
Is "safe-to-deploy" a pre-defined criteria?

Reply:
Yes, `safe-to-deploy` and `safe-to-run` are built in to cargo-vet: https://mozilla.github.io/cargo-vet/built-in-criteria.html